apps/homepage/src/lib/dictionaries/en.json

```diff
@@ -1,4 +1,7 @@
 {
   "title" : "HomePage",
-  "header" : "Header"
+  "header" : "Header",
+  "menuBtn1":"Home",
```

Comment:
Let's rename this to `homeLink`, `projectsLink`, etc... Because `menuBtn1` doesn't really tell what this is.